tests/components/samsungtv/test_config_flow.py

```diff
@@ -951,10 +962,13 @@ async def test_import_legacy_without_name(hass: HomeAssistant) -> None:
     assert result["data"][CONF_MANUFACTURER] == "Samsung"
     assert result["result"].unique_id is None
 
+    mock_setup_entry.assert_called_once()
     entries = hass.config_entries.async_entries(DOMAIN)
     assert len(entries) == 1
-    assert entries[0].data[CONF_METHOD] == METHOD_LEGACY
-    assert entries[0].data[CONF_PORT] == LEGACY_PORT
+    # METHOD / PORT failed during import
+    # They will get checked/set on setup
+    assert CONF_METHOD not in entries[0].data
+    assert CONF_PORT not in entries[0].data
```

Comment:
Slight change here: setup is now mocked so METHOD/PORT are no longer updated during the tests.

Reply:
I copied the test to `test_init.py` to ensure coverage is not affected.